android/src/org/kiwix/kiwixmobile/ZimFileSelectFragment.java

```diff
@@ -381,47 +415,4 @@ public View getView(int position, View convertView, ViewGroup parent) {
       ImageView favicon;
     }
   }
-
-  // This AsyncTask will scan the file system for files with the Extension ".zim" or ".zimaa"
-  private class RescanFileSystem extends AsyncTask<Void, Void, Void> {
-
-    @Override
-    protected void onPreExecute() {
-
-      super.onPreExecute();
-    }
-
-    @Override
-    protected Void doInBackground(Void... params) {
-      // Search file system for files
-      mFiles = new FileSearch().findFiles(context);
-
-      // Remove duplicate files
-      HashSet<LibraryNetworkEntity.Book> singularBooks = new HashSet<>();
-      singularBooks.addAll(mFiles);
-      mFiles.clear();
-      mFiles.addAll(singularBooks);
-
-      // Sort files
-      Collections.sort(mFiles, new fileComparator());
-
-      // Save files for quick access later
-      bookDao.saveBooks(mFiles);
```

Comment:
Database is now refreshed [here](https://github.com/kiwix/kiwix/pull/379/commits/77d2974bfcbced3dec55788fe02b62b13a25942f#diff-fa7f4e78c060f2453efcc57f82636b7dR228)